src/build.c

```diff
@@ -1057,7 +1057,7 @@ gboolean build_parse_make_dir(const gchar *string, gchar **prefix)
 	if (string == NULL)
 		return FALSE;
 
-	if ((pos = strstr(string, "Entering directory")) != NULL)
+	if (strstr(string, "Entering directory") != NULL)
```

Comment:
What about this, to avoid rescanning the beginning of the string:
```C
if ((pos = strstr(string, "Entering directory")) != NULL)
{
	// ...

	pos = strstr(pos, "/");   // instead of ... pos = strstr(string, "/");

	// ...
}
```

Reply:
And is there any reason not to use strchr instead of strstr?

Reply:
@andy5995 Are there any tangible benefits besides maybe saving a few processor cycles or avoiding some nested function call?

Reply:
@xiota 

> pos = strstr(pos, "/");   // instead of ... pos = strstr(string, "/");

Sounds good! I'll update the PR. Thanks!

> Are there any tangible benefits besides maybe saving a few processor cycles or avoiding some nested function call?

Why to not save maybe a few processor cycles or not avoid some nested function call? :)

Reply:
Upon further reflection... the change I suggested rescans the "Entering directory" portion of the string...  Should probably be (but haven't tested):
```C
pos = strchr(pos + 18, '/');
```
From search, the string looks like:
```
make[1]: Entering directory `/mnt/lfs/sources/gcc-build'
```